### ISSUE TITLE
Compatibility List

### ISSUE THREAD

**Issue by xunxky** [pytorch/pytorch#80857]
### 🚀 The feature, motivation and pitch

Have an easily to find (e.g. in the previous-versions page) compatibility list. If it has to be in the form of commands and comments ( just like in the previous-versions page), instead of a proper table, then this would be fine, too. 

However I believe many people wont know what version to use. and a proper tabulation would help. 

### Alternatives

wasting good nights sleep to find out it doesn't work, after jumping through all the hoops was not a good alternative. 

### Additional context

this is no "help request" or anything, but simply a feature request for a general professional table (or some list of arbitrary commands)

for context: 

Even though my pytorch version and "CUDA SDK version" are valid and below the max supported driver version as reported by smi (max = 11.4, installed 11.3) ( `pytorch==1.11.0 torchvision==0.12.0 torchaudio==0.11.0 cudatoolkit=11.3` ) pytorch slams the GC max supported "CUDA version" as 3.7 ... nothing you can actually find out until after the fact based on looking at the installation information

**Comment by dboggs95** [comment]
I like this idea. Can the table also include ROCm versions as well? If I can't do anything else to solve my problem trying to use an RX 580, I might try my hand at building PyTorch myself, and I'd like to know which versions of the code to use.

**Comment by ezyang** [comment]
Yeah, this is a good idea. If y'all want to crowdsource the list here, that's probably the easiest way to make this happen
